### DISCUSSION TITLE
Need More toilet quests

### DISCUSSION THREAD

**Discussion by timothywashere** [streetcomplete/StreetComplete#4057] [Ideas]
For people of a certain age it is obvious that OSM needs more information about toilets. SC could help with these types of extra quests. I have checked the wiki and they're not on it.

What are the opening hours of this toilet ? (when stand alone) Where I live most public toilets close at a certain time.
Does this petrol station have a public toilet? 
Does this supermarket have a public toilet?
Does this train station have a public toilet?
Does this coach station have a public toilet?

In the UK all of the above have a decent chance of having a public toilet.

**Comment by matkoniecz** [comment]
See https://github.com/streetcomplete/StreetComplete/issues/391 - especially since https://github.com/streetcomplete/StreetComplete/issues/391#issuecomment-441372108

It was discussed years ago so it may be useful to revisit it
